### ISSUE TITLE
Can't use any system themes

### ISSUE THREAD

**Issue by gardotd426** [linuxdeepin/developer-center#1723]
## Describe the bug
In the newest updated version of the desktop environment (running on Arch/Manjaro), the Control Center is a hot mess. But because of the request to only list one issue per post, this is about theming.

## To Reproduce

1. Make sure you're running the latest version of the Deepin package group. I'll list the current versions of all packages at the bottom of the post. 
2. Open the new Control Center, go to the "Personalization" section.
3. There are no system themes, only "Light," "Dark," and "Auto," whatever the hell Auto is. However, icon themes and cursor themes all show up as expected. Screenshots:

## Expected behavior
The prior version, with the side-panel slide-out Control Center, under Personlization, it showed all of your system-installed themes, from which you could select. On the new one, icon themes show up as expected, just like before, but the system/application themes are all gone. This is true whether using deepin-mutter or deepin-kwin. Even though, at least while using deepin-mutter, you CAN successfully get the system theme to change to any other theme you have installed by going and editing the schemas in dconf-editor and entering in the name of your theme. But without doing that, you only get "deepindark," "deepin" (light theme), or "Auto." Which again, Auto does absolutely nothing from what I can tell, I was hoping it was just a way to use a manually set theme, but setting it to Auto and then setting the theme with something like lxappearance (for gtk with deepin-mutter) or qt5ct (with deepin-kwin), but no, it just does nothing. 

## Screenshots
Working Icon Theme selection:

![deepinbug2](https://user-images.githubusercontent.com/54234607/81497253-4ee59680-928b-11ea-9bb5-0d5ee2393a6d.png)

No system theme available:

![deepinbug1](https://user-images.githubusercontent.com/54234607/81496606-c4029d00-9286-11ea-85cc-1c2289e11662.png)

### Distro & Version
Manjaro Linux 20.0.1, 

### Related package version
deepin package versions are as follows:
```
pacman -Q | egrep -i "deepin|dde"


deepin-account-faces 1.0.11-1.1
deepin-anything 5.0.1-2
deepin-anything-dkms 5.0.1-2
deepin-api 5.1.11.1-1
deepin-boot-maker 5.4.8-1
deepin-calculator 5.5.9-1
deepin-calendar 5.6.2-1
deepin-clone 5.0.3-1
deepin-clutter 1.26.2-2
deepin-cogl 1.22.5-3
deepin-control-center 5.1.0.5-1
deepin-daemon 5.9.4.2-1
deepin-desktop-base 3:2019.07.10-1
deepin-desktop-schemas-manjaro 5.4.0099.0-1
deepin-dock 5.1.0.8-1
deepin-draw 5.8.0.15-2
deepin-editor 5.6.7-2
deepin-file-manager 1:5.1.1.25-1
deepin-gtk-theme 17.10.11-1
deepin-icon-theme 2020.05.09-1
deepin-iconthemes-manjaro 20170918-1
deepin-image-viewer 5.6.3.2-1
deepin-launcher 5.1.0099.0-1
deepin-menu 5.0.1-1
deepin-metacity 3.22.24-1
deepin-movie 1:5.7.6.29-1
deepin-music 6.0.1.8-1
deepin-mutter 3.20.38-4
deepin-network-utils 5.1.0.0-1
deepin-picker 5.0.6-1
deepin-polkit-agent 5.1.0-1
deepin-polkit-agent-ext-gnomekeyring 0.1.0-1
deepin-qt-dbus-factory 5.1.0.0-1
deepin-qt5dxcb-plugin 5.0.11-1
deepin-qt5integration 5.1.0-1
deepin-screen-recorder 5.8.0.11-1
deepin-screensaver 0.0.10-1
deepin-screensaver-pp 0.0.4-1
deepin-session-shell 5.0.0098.0-2
deepin-session-ui 5.1.0099.4-1
deepin-shortcut-viewer 5.0.2-1
deepin-sound-theme 15.10.4-1
deepin-system-monitor 5.6.1-1
deepin-terminal 5.0.4.1-1
deepin-turbo 0.0.3-1
deepin-voice-recorder 5.0.0-1
deepin-wallpapers 2:1.7.7-1
deepin-wm 1.9.38-4
manjaro-deepin-settings 20191111-1
plymouth-theme-manjaro-deepin-circle 1.0-4
startdde 5.3.0.1-2
```     

### Additional context
I know this is still in beta, but it seems there are a lot of bugs/missing features that were present in the previous version and also on every other linux desktop environment there is. There have been a lot of improvements, like being able to set refresh rate for monitors in the control center, but there are also still a ton of bugs. I still have three or more to file, but it's late and I need some rest, I'll file the rest later today. But yeah, the theming is all jacked up, and it makes it really hard to make the desktop look good at all.

**Comment by philmmanjaro** [comment]
@gardotd426 it seems it is designed as it should be. Three options to change the way Deepin should look like: light, dark and auto, which might switch automatically if you switch one application. It is clear why general theming got removed from v20. It is unneeded and creates more trouble for the developers. Remember that Deepin is for paid customers and more or less designed to be simple. Adding more issues with the option to allow themes other than those shipped by Deepin itself is logical and needed to get Deepin more stable. In short I see no issues with removing the support of system themes, aka regular themes you might see for GTK and Qt.

**Comment by Y0ngg4n** [comment]
Duplicate of https://github.com/linuxdeepin/developer-center/issues/1727

**Comment by gardotd426** [comment]
@Y0ngg4n no, it's not. 1727 is for the three available DEEPIN themes in the control center not working. That has nothing to do with this issue. This issue is that all the other themes one has installed don't show up. On Deepin 15, if you went to Personalization, under "Theme" you could select from any theme you had installed, like Adwaita, Arc, Breeze, Evopop, Papirus-Dark, etc. But on the new Control Center, you get "Light," "Dark," and "Auto," and none of the other themes you have installed show up. 

#1727 is about how even the "Light," "Dark," and "Auto" don't work on Deepin apps, not about the loss of all other themes.

**Comment by Y0ngg4n** [comment]
@gardotd426 Ah ok sorry my mistake.

**Comment by meiyixiang** [comment]
@justforlxz I think it's a BUG. Take a look
